saga/fx/cancel.ts

```diff
@@ -0,0 +1,9 @@
+import { action } from "../../instructions.ts";
+import type { Frame } from "../../types.ts";
+
+export function cancel(frame: Frame) {
```

Comment:
I wonder if there is a way to do this with `Task#halt()`  or `Scope#close()` (the latter of which did not exist yesterday 😀)